eldev.el

```diff
@@ -3137,8 +3159,12 @@ least one warning."
       (eldev-print "Nothing to do"))))
 
 (defun eldev-lint-default-p (linter)
-  (and (or (eq eldev-lint-default t) (memq linter eldev-lint-default))
-       (not (memq linter eldev-lint-default-excluded)) (not (memq linter eldev-lint-disabled))))
+  (and (eldev-pcase-exhaustive eldev-lint-default
+         (`t t)
+         (:default (assq :default (assq linter eldev--linters)))
```

Comment:
As it stands, I get the following if I try running it:

    ~/eldev$ eldev lint
    Wrong type argument: listp, (elisp . eldev-linter-elisp)